go/types.go

```diff
@@ -238,6 +241,7 @@ type LogEntry struct {
 	Leaf     MerkleTreeLeaf
 	X509Cert *x509.Certificate
 	Precert  *Precertificate
+	JSONData []byte
```

Comment:
Wouldn't that be better as an `interface{}` here? Your call, I'm not really sure where this gets used...